codebase/superdarn/src.bin/tk/tool/map_addhmb.1.17/map_addhmb.c

```diff
@@ -74,7 +74,7 @@ int main(int argc,char *argv[])
   int tme;
   int yrsec=0;
   int i,j;
-  int s;
+  int s=0;
```

Comment:
Why do you set s = 0 but not i,j on line 76 or others like x on line 89?

Reply:
Funny thing there was a warning for it:
map_addhmb.c:268:11: warning: ‘s’ may be used uninitialized in this function [-Wmaybe-uninitialized]

Maybe we should look at the warnings more closely :p